lib/metalsmith.js

```diff
@@ -22,12 +23,42 @@ const views = [
   paths.govukfrontend
 ]
 
+const coloursAppliedSCSS = scssToJson('./node_modules/@govuk-frontend/globals/settings/_colours-applied.scss', {
+  dependencies: [{path: './node_modules/@govuk-frontend/globals/settings/_colours-palette.scss'}]
+})
+
+const coloursPaletteSCSS = scssToJson('./node_modules/@govuk-frontend/globals/settings/_colours-palette.scss')
+
+// combine both SCSS data objects - some Main colours are only in Palette
+
```

Comment:
Nit: For consistency with rest of file, delete space from below comment so it's clear it relates to the one below